test/src/unit_tests/cases/LPT_scheduling.test.cxx

```diff
@@ -1,3 +1,4 @@
+#include <algorithm>
```

Comment:
This include should be added to `LPT_scheduling.hxx` (which also contains `std::sort()`).